### PR TITLE
1 25 22 review

### PR DESCRIPTION
**Pull request by astoycos**

### CODE REVIEW

File: keps/sig-network/2091-admin-network-policy/README.md [hunk 1]
```diff
@@ -199,55 +201,42 @@ admins explicitly by the cluster admin with the use of `Pass` rules.
...
 
 #### Rule Names
 
 In order to help future proof the ANP api, a built in mechanism to identify each
 allow/deny/pass rule is required. Such a mechanism will help administrators organize 
 and identify individual rules within an AdminNetworkPolicy resource.
 We propose to introduce a new string field, called `name`, in each `AdminNetworkPolicy`
-ingress/egress rule. The `name` of a rule must be unique within the same ANP. However,
-two ANP instances could have rules with same names.
+ingress/egress rule. Currently the `name` of a rule is optional and does need to
```
Comment (Dyanngg) [inline keps/sig-network/2091-admin-network-policy/README.md:238]: you mean `does` or `does not`? If it does need to be unique within a policy (we will need to specify the scope here), I guess we'll have to say there's no way in the API validation that we can enforce that, and implementations would need to take care of the validation

Comment (astoycos) [reply keps/sig-network/2091-admin-network-policy/README.md:238]: I meant "does not" because there's no way to validate it... I'll try and add a note

Comment (astoycos) [reply keps/sig-network/2091-admin-network-policy/README.md:238]:
```
Currently the `name` of a rule is optional and does not need to 
be unique, due to the fact that there's no easy way to validate this in-tree. It 
is suggested that implementations provide a mechanism to validate any name 
duplications.  
```

Comment (astoycos) [reply keps/sig-network/2091-admin-network-policy/README.md:238]: Is something like this alright?

Comment (Dyanngg) [reply keps/sig-network/2091-admin-network-policy/README.md:238]: Sounds good to me

Comment (astoycos) [reply keps/sig-network/2091-admin-network-policy/README.md]: Fixed

**Pull request merged**